### ISSUE TITLE
search disconnected external hard drives 

### ISSUE THREAD

**Issue by He1my** [cboxdoerfer/fsearch#358]
I have multiple external USB drives used for archival, I used to use Fsearch to index the drives and search their files when disconnected, it tells me easily which drive I needed to connect to get the files I needed, 
I just updated Fsearch using the AUR package, "0.2alpha (AUR-devel)"
I noticed a few changes that make me unable to exactly do what I used to do before 
- the new version doesn't recognize the old databases located in (~/.config/fsearch/), is there a way to "import" the databases from the old version format to the new one? I could connect the external drives again and index them, but I have the file list of some drives that failed in the old database and I can't access them in the new version now!

- the database file is now a single file (~/.local/share/fsearch/fsearch.db) instead of one file per search location as it is used to 
is there a way to have multiple db files per location as it used to? this was very helpful, especially when I needed to have a sort of a backup of the external drive files list, and it was very useful when I had drives that failed what was lost exactly from the Fsearch results

- is there a way in the new version to **not** update a location but still have it searchable? similar to the update column

**Comment by cboxdoerfer** [comment]
> the new version doesn't recognize the old databases located in (~/.config/fsearch/), is there a way to "import" the databases from the old version format to the new one? I could connect the external drives again and index them, but I have the file list of some drives that failed in the old database and I can't access them in the new version now!

No, it's not possible to import the old database format. It's not difficult to implement, so if someone wants to work on that I'll accept pull requests. But since the old database format was only used before the first stable 0.1 release I won't implement it myself, as there are much more important things on my roadmap.

> the database file is now a single file (~/.local/share/fsearch/fsearch.db) instead of one file per search location as it is used to
is there a way to have multiple db files per location as it used to? this was very helpful, especially when I needed to have a sort of a backup of the external drive files list, and it was very useful when I had drives that failed what was lost exactly from the Fsearch results

No, there isn't and most likely won't be an option to have separate database files for each location. The old design made the new fast sort index much more complicated to implement and would have significantly increased database load times, i.e. application startup.

However it will be possible to define a custom database location at startup in the future and it will be possible to dump the database (either fully or only selected locations) to a file or console.

> is there a way in the new version to not update a location but still have it searchable? similar to the update column

Not at the moment. This feature is planned to come back with the 0.3. release.
